### PR TITLE
FUSETOOLS2-469 - have deterministic test checking for AtlasMap opened

### PR DESCRIPTION
**Pull request by apupier**
now that the content of index.html is retrieve dynamically, we need to
provide a conditional wait in tests to ensure that it had time to load
the content of AtlasMap UI

Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by apupier** [comment]
one success https://travis-ci.org/github/jboss-fuse/vscode-atlasmap/builds/694673101

**Comment by apupier** [comment]
another success https://travis-ci.org/github/jboss-fuse/vscode-atlasmap/builds/694676420

and seems the link between Travis and GitHub is not working

File: src/test/command.start.test.ts [hunk 1]
```diff
@@ -280,8 +281,10 @@ testUtils.BROWSER_TYPES.forEach(function (browserConfig) {
 	});
 });
 
-
-function checkBorderStyleAvailable() {
-	expect(atlasMapWebView.default.currentPanel._panel.webview.html, "HTML doesn't contain url the <title>AtlasMap Data Mapper UI</title>").to.contain('<title>AtlasMap Data Mapper UI</title>');
+function checkContainsAtlasMapTitle() {
+	const expectedAtlasMapTitle = '<title>AtlasMap Data Mapper UI</title>';
+	waitUntil(() => atlasMapWebView.default.currentPanel._panel.webview.html.includes(expectedAtlasMapTitle));
```
Comment (lhein) [inline src/test/command.start.test.ts:286]: is there no timeout?

Comment (apupier) [reply src/test/command.start.test.ts:286]: default is 5s, see https://www.npmjs.com/package/async-wait-until#supported-arguments

Comment (lhein) [reply src/test/command.start.test.ts:286]: 5s sounds fine

**Pull request merged**